proposals/dictionary-expressions.md

```diff
@@ -165,47 +165,47 @@ From [Open Question 3](#open-question-3), we will support *dictionary elements*
 2. Use applicable instance indexer if available; otherwise report an error during construction (or conversion?).
 3. Use C#12 initialization always.
 
+## Dictionary types
+
+The following are considered *dictionary types*:
+
+* A *type* with an appropriate *[create method](#create-methods)* where:
+  * The *iteration type* is `KeyValuePair<TKey, TValue>` determined from a `GetEnumerator` instance method or enumerable interface.
+  * The *type* has an instance *indexer* where:
+    * The indexer has a single parameter with an identity conversion from the parameter type to `TKey`.
```

Comment:
Shouldn't it be 

```suggestion
    * The indexer has a single parameter with an identity conversion from `TKey` to the parameter type.
```

Because the goal is for `TKey` to be passable to the indexer